TFT/src/User/API/Settings.c

```diff
@@ -244,7 +244,11 @@ void setupMachine(FW_TYPE fwType)
   }
 
   mustStoreCmd("M503 S0\n");
-  mustStoreCmd("M82\n");  // Set extruder to absolute mode
+  if (infoSettings.hotend_count > 0)
```

Comment:
please use an indentation of 2 chars in all the code and never use tabs

Reply:
Have you ever used clang-format? It is really nice to be able to just bulk fix a ton of formatting across an entire project. You can also add it to the CI, so that every PR has to pass the formatting first (but it can be a bit annoying to new users).

Reply:
yes I know but the project is open source (so even for people with not so much experience) and everyone most of the time has a different/preferred coding style. At the end we apply some basic common styles acceptable by everyone.